tests/integration/piv_test.py

```diff
@@ -1661,3 +1658,245 @@ def test_piv_re_routed_in_the_end(mock_rabbitmq):
+
+
+def _assert_db_piv_trip_creation():
+    with app.app_context():
+        assert RealTimeUpdate.query.count() == 1
+        assert TripUpdate.query.count() == 1
+        db_trip = TripUpdate.query.first()
+
+        assert db_trip.status == ModificationType.add.name
+        assert db_trip.effect == TripEffect.ADDITIONAL_SERVICE.name
+        assert len(db_trip.stop_time_updates) == 6
+
+        added_st = db_trip.stop_time_updates[0]
+        assert added_st.arrival_status == ModificationType.none.name
+        assert added_st.arrival == datetime(2020, 10, 22, 20, 34)
+        assert added_st.departure_status == ModificationType.add.name
+        assert added_st.departure == datetime(2020, 10, 22, 20, 34)
+
+        added_st = db_trip.stop_time_updates[1]
+        assert added_st.arrival_status == ModificationType.add.name
+        assert added_st.arrival == datetime(2020, 10, 22, 20, 35)
+        assert added_st.departure_status == ModificationType.add.name
+        assert added_st.departure == datetime(2020, 10, 22, 20, 35, 30)
+
+        added_st = db_trip.stop_time_updates[2]
+        assert added_st.arrival_status == ModificationType.add.name
+        assert added_st.arrival == datetime(2020, 10, 22, 20, 47)
+        assert added_st.departure_status == ModificationType.add.name
+        assert added_st.departure == datetime(2020, 10, 22, 20, 48)
+
+        added_st = db_trip.stop_time_updates[3]
+        assert added_st.arrival_status == ModificationType.add.name
+        assert added_st.arrival == datetime(2020, 10, 22, 21, 16)
+        assert added_st.departure_status == ModificationType.add.name
+        assert added_st.departure == datetime(2020, 10, 22, 21, 17)
+
+        added_st = db_trip.stop_time_updates[4]
+        assert added_st.arrival_status == ModificationType.add.name
+        assert added_st.arrival == datetime(2020, 10, 22, 21, 19)
+        assert added_st.departure_status == ModificationType.add.name
+        assert added_st.departure == datetime(2020, 10, 22, 21, 20)
+
+        added_st = db_trip.stop_time_updates[5]
+        assert added_st.arrival_status == ModificationType.add.name
+        assert added_st.arrival == datetime(2020, 10, 22, 21, 25)
+        assert added_st.departure_status == ModificationType.none.name
+        assert added_st.departure == datetime(2020, 10, 22, 21, 25)
+
+
+def test_piv_trip_creation(mock_rabbitmq):
+    piv_feed = _get_stomp_20201022_23188_add_trip_fixture(plan_transport_source="PTP")
+    res = api_post("/piv/{}".format(PIV_CONTRIBUTOR_ID), data=ujson.dumps(piv_feed))
+    assert "PIV feed processed" in res.get("message")
+
+    _assert_db_piv_trip_creation()
+
+
+def test_piv_trip_creation_delayed_0min(mock_rabbitmq):
+    piv_feed = _get_stomp_20201022_23188_add_trip_fixture()
+    disruption = DisruptionTuple(type="RETARD_PROJETE", texte="Absence inopinée d'un agent")
+    modification = ModificationTuple(motif=None, statut=None)
+    retard = {"duree": 0, "dureeInterne": 3}
```

Comment:
Can you change it to have a case which is more tricky?
```suggestion
    retard = {"duree": 5, "dureeInterne": 6}
```
This should still give a `0` delay as this is added stop, so only `dateHeureReelle` is considered.

Reply:
It conforms with test_piv_trip_creation_delayed_5min test